src/mlpack/methods/ann/activation_functions/hard_sigmoid_function.hpp

```diff
@@ -0,0 +1,103 @@
+/**
+ * @file hard_sigmoid_function.hpp
+ * @author Bishwa Karki
+ *
+ * Definition and implementation of the hard sigmoid function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_HARD_SIGMOID_FUNCTION_HPP
+#define MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_HARD_SIGMOID_FUNCTION_HPP
+
+#include <mlpack/prereqs.hpp>
+#include <algorithm>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * The hard sigmoid function, defined by
+ *
+ * @f{eqnarray*}{
+ * if x < -2.5 \\
```

Comment:
This is no valid latex code, can you check the code.

Reply:
Ack, should've paid more attention here. What is the standard way of checking for LaTeX code in pull requests?

Reply:
One option might be to look at the latex job output, not ideal but it works.

Reply:
Yeah, if you have LaTeX installed, `make doc` should give an error.

Reply:
Hmm, I'll have to pull in the changes for that. I wonder why LaTeX Documentation check shows a valid build though?

Reply:
Doxygen is still producing some output it just looks wrong.